### DISCUSSION TITLE
Editor doesn't detect songs' BPM

### DISCUSSION THREAD

**Discussion by federicochiesa** [ppy/osu#23062] [Q&A] [answered]
### Type

Game behaviour

### Bug description

If you add a song in the editor it doesn't scan the BPM or give me the waveform and  the BPM remains at 60.

### Version

2023.326.1

### Logs

[database.log](https://github.com/ppy/osu/files/11110171/database.log)
[input.log](https://github.com/ppy/osu/files/11110172/input.log)
[network.log](https://github.com/ppy/osu/files/11110173/network.log)
[performance.log](https://github.com/ppy/osu/files/11110174/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11110175/runtime.log)

**Comment by peppy** [comment, marked as answer]
It's not supposed to. Finding the BPM is part of the fun!
